pennylane/optimize/momentum.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, stepsize=0.01, momentum=0.9):
         self.momentum = momentum
         self.accumulation = None
 
-    def apply_grad(self, grad, x):
+    def apply_grad(self, grad, args):
```

Comment:
(remember to update the docstring below!)

Reply:
and again, now fixed.